docs/content/preview/quick-start/_index.md

```diff
@@ -204,7 +205,7 @@ macOS Monterey enables AirPlay receiving by default, which listens on port 7000.
 
 {{< /note >}}
 
-After the cluster has been created, clients can connect to the YSQL and YCQL APIs at http://localhost:5433 and http://localhost:9042 respectively. You can also check `~/var/data` to see the data directory and `~/var/logs` to see the logs directory.
+After the cluster has been created, clients can connect to the YSQL and YCQL APIs at `http://localhost:5433` and `http://localhost:9042` respectively. You can also check `~/var/data` to see the data directory and `~/var/logs` to see the logs directory.
```

Comment:
```suggestion
After the cluster has been created, clients can connect to the YSQL and YCQL APIs at <http://localhost:5433> and <http://localhost:9042> respectively. You can also check `~/var/data` to see the data directory and `~/var/logs` to see the logs directory.
```

Reply:
I put these in code rather than link because they are used in applications, and if you click and go via browser you don't see anything